dags/trigger_k8s_cronjob.py

```diff
@@ -31,9 +31,9 @@ def trigger_k8s_cronjob(cronjob_name, namespace):
   cronjob = get_cronjob(cronjob_name, namespace)
 
   if cronjob:
-    date_str = datetime.datetime.now().strftime("%Y%m%d%H%M%S.%f")
+    date_str = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
     # Change the name of the job to be created to show that it was manually created at time: date_str
-    cronjob.spec.job_template.metadata.name = str(cronjob.metadata.name + '-manual-' + date_str)[:50]
+    cronjob.spec.job_template.metadata.name = str(date_str + cronjob.metadata.name)
```

Comment:
you still need to truncate it though. I think that 63 characters should be the max, following the rules that is used in helm templates:
> We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).